### PR TITLE
FEAT - add `ajna-contracts`

### PR DESCRIPTION
**Pull request by halaprix**

### CODE REVIEW

File: packages/dma-contracts/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# @oasisdex/dma-contracts
+
+## RUN BEFORE USING DMA-CONTRACTS
+You need to create a symlink, to include `ajna-contracts` in `dma-contracts` for proper solc compilation.
+```bash
+/oasis-earn-sc/packages/dma-contracts$ ln -s ~/absolute_path_to_the_repository/oasis-earn-sc/packages/ajna-contracts/contracts/ajna ./contracts/
+```
```
Comment (paszkowskiDamian) [inline packages/dma-contracts/README.md:7]: Isn't lerna supposed to do it?

File: packages/abis/system/contracts/interfaces/dpm/IAccountGuard.sol/IAccountGuard.json [hunk 2]
```diff
@@ -1,4 +1,98 @@
 [
```
Comment (paszkowskiDamian) [inline packages/abis/system/contracts/interfaces/dpm/IAccountGuard.sol/IAccountGuard.json:1]: Shouldn't we use abis and contracts already deified? Or this is the issue with hardhat not being able to find Contract Factories?

File: packages/dma-contracts/README.md (hunk 1 above)
Comment (halaprix) [reply packages/dma-contracts/README.md:7]: it's an additional symlink to make hh compile contracts from `ajna-contracts` while running compile in `dma-contracts` . temporary solution, and after ajna release we will get rid of this package or at least not use it in `dma-contracts`

File: packages/abis/system/contracts/interfaces/dpm/IAccountGuard.sol/IAccountGuard.json (hunk 2 above)
Comment (halaprix) [reply packages/abis/system/contracts/interfaces/dpm/IAccountGuard.sol/IAccountGuard.json:1]: Let me check that tomorrow. For sure I've expanded the interfaces for guard and account, but the same contract sources are used.

**Review by paszkowskiDamian — APPROVED** [review]

**Pull request merged**